force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -288,8 +319,14 @@ public with sharing class FlowControlLoopsCollections {
     public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
-
-        return null; // Replace null with the variable you used to store the result
+         Decimal sum = 0;
+         for (Decimal money : moneyInWallet) {
+            sum += money;
+            if (sum > 40) { 
```

Comment:
Try adding a variable for the number 40 so it's not a magic number  https://en.wikipedia.org/wiki/Magic_number_(programming)